include/strprintf.h

```diff
@@ -78,6 +41,24 @@ class StrPrintf {
 
 		return result + fmt(remaining_format, args...);
 	}
+
+	template<typename... Args>
+	std::string fmt(const std::string& format,
+		const std::string& argument,
+		Args... args)
+	{
+		const char * arg = argument.c_str();
```

Comment:
Again, just curious: why put that pointer into a variable rather than simply passing it to `fmt`?

Also, in Newsboat ampersands and asterisks go right after the type (like with `string::string&` in this function's signature).

Reply:
There is no point, I just forgot to revert it.